src/Evolution/Systems/ScalarWave/System.hpp

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Why the extra newline?

Reply:
I removed the extra newline. Is that an issue?

Reply:
I must've been tired and thought the newline was being added...